plugins/ps/commands

```diff
@@ -70,7 +70,10 @@ case "$1" in
 
     ! (is_deployed $APP) && echo "App $APP has not been deployed" && exit 0
 
-    release_and_deploy "$APP" "$IMAGE_TAG"
+    if (is_app_running $APP); then
+        dokku ps:stop "$APP"
+    fi
+    dokku ps:start "$APP"
```

Comment:
Yes, that might be a good idea. The same goes for these instances:
https://github.com/bobvanderlinden/dokku/blob/fix-psrestart/plugins/ps/commands#L84
https://github.com/bobvanderlinden/dokku/blob/fix-psrestart/plugins/ps/commands#L61

Should these also be turned into functions?